dockerfiles/che/rhel.Dockerfile

```diff
@@ -32,8 +30,6 @@ RUN mkdir /logs /data && \
     chown -R user /home/user && \
     chmod -R g+rwX /home/user && \
     find /home/user -type d -exec chmod 777 {} \; && \
-    # set group write permission so that entrypoint.sh can update permissions once file is updated w/ new cert
-    chmod g+w /home/user/cacerts && \
```

Comment:
If we're still using `/home/user/cacerts` file as per https://github.com/eclipse/che/pull/18504/files#diff-0494d7f08e22baf0f7a6a136eb6f5a6a6532d3199d4edda56964dae41c18b22bR326 why do we no longer need to set perms on it?

Reply:
Because we do not copy it in dockerfile anymore.

Reply:
I see -- so the certs are copied, updated, etc. all as the anyuid user during entrypoint, not as a mix of docker/root and entrypoint/anyuid.